data/json/items/containers.json

```diff
@@ -1664,16 +1664,40 @@
         "description": "This waterproof phone case has a drawing on it that makes it look like an old English police call box.",
         "weight": 1
       },
+      {
+        "id": "black_smart_phone_case",
+        "name": { "str_sp": "black waterproof smartphone case" },
+        "description": "A waterproof case designed for a standard smartphone.  Will protect your smartphone from water damage entirely.  This one is black.",
```

Comment:
I'd suggest rephrasing these to just "a <color> waterproof case". The "This one is <color>" text feels strange if you're only looking at one.

I'd also consider moving "Will protect your smartphone from water damage entirely." to a flag to prevent duplication. Even better if it's a generic one for waterproofing, but I haven't looked to see if we have one of those.

I do love seeing all these variants, but I'm a little concerned about the duplication of translations this is going to cause. I'm not saying you need to make any of these changes or even be concerned with this now, but we'll probably want to come up with a better solution before we get too deep into variants.

Reply:
`I'd also consider moving "Will protect your smartphone from water damage entirely." to a flag to prevent duplication. Even better if it's a generic one for waterproofing, but I haven't looked to see if we have one of those.`

You can't flag variants.  They are all exactly the same except for descriptions.   I could remove that line of text but it pre-existed the variants to prevent it from being a newbie trap where it's effect is hidden in the pocket data.

Reply:
I'll make a note to review if it makes sense to use a flag on the base item instead when I get around to implementing flags for some known duplicates. It does seem a bit silly if it's only being used once, but if it reduces the same string getting re-translated and makes sense to be moved out the description, then I'll probably still do it.

> only thing that get's translated as a new translation is the description and name

Yes, but each description gets translated as a whole, so each time the same sentence appears in different descriptions, it needs to be translated again. This doesn't sound so bad when we're talking about a handful of descriptions, but it's how we've ended up with the same (or near same) text on 50-100 descriptions over time.

For now I wouldn't worry about it. I'll review it for flag usage when I get to that.